.github/workflows/test.yml

```diff
@@ -94,11 +68,27 @@ jobs:
         name: junit-${{ github.run_id }}
         path: '**/_artifacts/**.xml'
 
-    - name: Submit code coverage to Coveralls
-      uses: coverallsapp/github-action@master
+    - name: Publish Test Report
+      uses: mikepenz/action-junit-report@v2
```

Comment:
it is failing
```

Build❌ Failed to create checks using the provided token. (HttpError: Resource not accessible by integration)
--
Build⚠️ This usually indicates insufficient permissions. More details: https://github.com/mikepenz/action-junit-report/issues/23


```

Reply:
Yup. Need to run this on pull_request_target, not pull_request. Interesting.